app/templates/userProfile.html

```diff
@@ -1,13 +1,93 @@
-{% endfor %}
+  <div class="container py-5 h-100">
+    <div class="row d-flex justify-content-center align-items-center h-100">
+      <div class="col col-lg-6 mb-4 mb-lg-0">
+        <div class="card mb-3" style="border-radius: .5rem;">
+          <div class="row g-0">
+            <div class="col-md-4 gradient-custom text-center text-white" style="border-top-left-radius: .5rem; border-bottom-left-radius: .5rem;">
+              <img
+                src="https://mdbcdn.b-cdn.net/img/Photos/new-templates/bootstrap-chat/ava1-bg.webp"
+                alt="Avatar"
+                class="img-fluid my-5"
+                style="width: 80px;"
+              />
+              <h5>adminone</h5>
+              <p>admin1</p>
 
+              <!-- {% if current_user.is_authenticated and current_user.role == 'staff' %}
+              <a href="{{ url_for('user.update', user_id=user.user_id) }}" type="button" class="btn btn-info">Update Profile</a>
+              {% endif %} -->
+
+              <!-- {% for attribute, value in user.__dict__.items() %}
+              <br /> {{ attribute }}::{{ value }}
+              {% endfor %} -->
+
+            </div>
+            <div class="col-md-8">
+              <div class="card-body p-4">
+                <h6>User Information</h6>
+                <hr class="mt-0 mb-4">
+                <div class="row pt-1">
+
+                  <div class="col-6 mb-3">
+                    <h6>First name</h6>
+                    <p class="text-muted">Irene</p>
+                  </div>
+
+                  <div class="col-6 mb-3">
+                    <h6>Last name</h6>
+                    <p class="text-muted">Zheng</p>
+                  </div>
+
+                  <div class="col-6 mb-3">
+                    <h6>email</h6>
+                    <p class="text-muted">hello@gmail.com</p>
+                  </div>
+
+                  <div class="col-6 mb-3">
+                    <h6>username</h6>
+                    <p class="text-muted">adminone</p>
+                  </div>
+
+                  {% if current_user.is_authenticated and current_user.role == 'admin' %}
+                  <div class="col-6 mb-3">
+                    <h6>role</h6>
+                    <p class="text-muted">admin</p>
+                  </div>
+                  {% endif %}
+
+                  {% if current_user.is_authenticated and current_user.role == 'admin' %}
+                  <div class="col-6 mb-3">
+                    <h6>last_login</h6>
+                    <p class="text-muted">active</p>
+                  </div>
+                  {% endif %}
+
+                  {% if current_user.is_authenticated and current_user.role == 'admin' %}
+                  <div class="col-6 mb-3">
+                    <h6>create_at</h6>
+                    <p class="text-muted">active</p>
+                  </div>
+                  {% endif %}
+
+                  {% if current_user.is_authenticated and current_user.role == 'admin' %}
```

Comment:
Those `div` tag for admin can share the `if statement`. To reduce the redundancy, please remove this.